cli/src/cli/index.ts

```diff
@@ -80,7 +80,7 @@ export const runCli = async () => {
      * If any of the following option-flags are provided, we skip prompting
      */
     .option(
-      "--tailwind <tailwind>",
+      "--tailwind <boolean>",
```

Comment:
These are actually incorrect, the value must match the key to evaluate to true

`--tailwind=tailwind => using tailwind`
`--tailwind => nope`
`--tailwind=true => nope`

It should all be redone, it was all done in a bit of a rush and it's not very clean